src/core/State.h

```diff
@@ -74,12 +76,12 @@ class State
     State();
     /**
      * Constructor
-     * @param contentWindows a list of contentWindows to serialize
+     * @param displayGroup the DisplayGroup to serialize
      */
-    State( const ContentWindowPtrs& contentWindows );
+    State( const DisplayGroupPtr displayGroup );
 
     /** Get the content windows */
-    const ContentWindowPtrs& getContentWindows() const;
+    ContentWindowPtrs& getContentWindows() const;
```

Comment:
Same note here, if the returned reference is not const the method should not be const